src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -668,7 +708,9 @@ public void TestPriorityClassWindows()
         public void TestInvalidPriorityClass(ProcessPriorityClass priorityClass)
         {
             var process = new Process();
-            Assert.Throws<InvalidEnumArgumentException>(() => process.PriorityClass = priorityClass);
+            var exception = Record.Exception(() => process.PriorityClass = priorityClass);
```

Comment:
@ViktorHofer This shouldn't repro anymore, this was fixed, please revert this change. All the process tests passed in the latest outerloop [run ](https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_netfx_windows_nt_debug/lastCompletedBuild/consoleFull).

```
System.Diagnostics.Process.Tests  Total: 206, Errors: 0, Failed: 0, Skipped: 0, Time: 12.211s
```